src/main/java/org/apache/cassandra/sidecar/routes/AbstractHandler.java

```diff
@@ -93,6 +93,7 @@ public void handle(RoutingContext context)
         catch (Exception exception)
         {
             processFailure(exception, context, host, remoteAddress, null);
+            return;
```

Comment:
Instead of `return` from the catch block, how about moving the `handleInternal` call inside the try block? So that it catches any exception throw during `handleInternal`.

Reply:
If `handlerInternal` moved to try block, then processFailure will end up handling expections of `handleInternal` method too. As per my understand, processFailure is only for processing the failures of request parsing/exptract params. If `processFailure` starts handling `handlerInternal` exceptions too, too then isn't it a change of behavior for it?

Reply:
I believe the intent here is to have a common catch block for any exception during `handle`. Moving `handlerInternal` inside the try block is the desired correction of the behavior. 
Maybe @frankgh can comment here, as you are the original author of the file.

Reply:
Yeah, `processFailure` is intended to process errors during the handleInternal method. So Yifan's suggestion makes sense

Reply:
https://github.com/apache/cassandra-sidecar/blob/trunk/src/main/java/org/apache/cassandra/sidecar/routes/AbstractHandler.java#L168 made me to think that it is only parsing params. Making the changes and updating the description too.

Reply:
@frankgh @yifan-c pushed the changes. [CI](https://app.circleci.com/pipelines/github/nvharikrishna/cassandra-sidecar/5/workflows/1f2da381-f29d-479c-8e56-a0eb157b3526) passed.